src/script/script.js

```diff
@@ -130,9 +140,18 @@ $(document).ready(function(){
     // END SECTION 6 and 10 //
 
     // SECTION OPENMAP //
-    $('.js-click-map').one('click', function () {
-      $('.js-map').addClass('opened');
-      $('.js-openmap-text').fadeOut(500)
+    $('.js-click-map').on('click', function () {
+        $('.js-map').addClass('opened');
+        $('.js-openmap-text').fadeOut(500);
+        $('.js-click-map').hide();
+        $('.js-close-map').show();
+    });
+
+    $('.js-close-map').on('click', function() {
+        $('.js-map').removeClass('opened');
+        $('.js-openmap-text').fadeIn(500);
```

Comment:
you don't need to hide `openmap-text` because it lies in `js-close-map` container and you closes it. So this line is unneeded and line 145 too